openedx/core/djangoapps/content/course_overviews/signals.py

```diff
@@ -0,0 +1,26 @@
+"""
+Signal handler for invalidating cached course overviews
+"""
+from celery.task import task
+from django.dispatch.dispatcher import receiver
+
+from xmodule.modulestore.django import SignalHandler
+
+from .models import CourseOverview
+
+
+@task()
+def _delete_entry(course_key):
```

Comment:
Let's remove this from being a separate celery task since it's just a delete operation.